### ISSUE TITLE
How to send Diagnosis.AnatomicalLocationSite

### ISSUE THREAD

**Issue by anthonygalba** [drajer-health/uds-plus#12]
How do we send the Anatomical Location Site QDM attribute on the UDS+ Diagnosis resource?
I don't see any element on the resource that matches that attribute, and it's also not mentioned on the QDM to FHIR crosswalk (https://fhir.org/guides/hrsa/uds-plus/dataelements.html).

That attribute is used in the logic for the Table 6B Line 11a (CMS-125) Breast Cancer Screening quality measure when determining whether there is a diagnosis for a right or left mastectomy.
Perhaps there needs to be an extension added for anatomical location site that can send the SNOMED codes used by the quality measure?

**Comment by nbashyam** [comment]
Anthony,

The De-identified Procedure resource has a body site, which can be used to indicate the above data element. Please let us know if this is not sufficient.
https://fhir.org/guides/hrsa/uds-plus/StructureDefinition-uds-plus-procedure.html

**Comment by anthonygalba** [comment]
Yes, I see that for procedures, but CMS-125 specifically uses Anatomical Location Site on a Diagnosis template, which should be sent as a UDS+ Diagnosis resource.

Are you saying we should be sending this diagnosis data as a procedure for this one specific case? Because that is definitely diverging from US Core and would need some special handling.

I'd also like to point out the value set used in the binding on Procedure Body Site does not actually contain the two SNOMED codes used for anatomical location site by CMS-125.
